src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -520,8 +524,8 @@ export class Tv2SplitScreenActionFactory {
   private createInsertLastVideoClipToInputActions(blueprintConfiguration: Tv2BlueprintConfiguration): Tv2SplitScreenInsertLastVideoClipInputAction[] {
     const actions: Tv2SplitScreenInsertLastVideoClipInputAction[] = []
     for (let inputIndex = 0; inputIndex < NUMBER_OF_SPLIT_SCREEN_BOXES; inputIndex++) {
-      actions.push(this.createInsertLastVideoClipToInputAction(blueprintConfiguration, inputIndex, Tv2AudioMode.FULL))
-      actions.push(this.createInsertLastVideoClipToInputAction(blueprintConfiguration, inputIndex, Tv2AudioMode.VOICE_OVER))
+      actions.push(this.createInsertLastVideoClipToInputAction(blueprintConfiguration, this.createDisplayIndex(inputIndex), Tv2AudioMode.FULL))
+      actions.push(this.createInsertLastVideoClipToInputAction(blueprintConfiguration, this.createDisplayIndex(inputIndex), Tv2AudioMode.VOICE_OVER))
```

Comment:
This doesn't solve it for the `InsertLastVideoClipAction`.
You have to call `createDisplayIndex` within `createInsertLastVideoClipToInputAction` method since the parameter `inputIndex` in that method is both given to the metadata and as the name. And as we talked about, the name should be one higher than the one given to the metadata :)